homeassistant/components/jvc_projector/__init__.py

```diff
@@ -0,0 +1,79 @@
+"""The jvc_projector integration."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from jvcprojector import JvcProjector, JvcProjectorAuthError, JvcProjectorConnectError
+
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    EVENT_HOMEASSISTANT_STOP,
+    Platform,
+)
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+
+from .const import DOMAIN
+from .coordinator import JvcProjectorDataUpdateCoordinator
+
+if TYPE_CHECKING:
+    from homeassistant.config_entries import ConfigEntry
+    from homeassistant.core import Event, HomeAssistant
```

Comment:
We normally only guarg for TYPE_CHECKING where it generates circular references. In this case it's better to remove the guard